src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -118,8 +118,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         // load any messages from templates
         $extractedCatalogue = new MessageCatalogue($input->getArgument('locale'));
         $output->text('Parsing templates');
+        $prefix = $input->getOption('prefix');
```

Comment:
Maybe I am misunderstanding this, but `$prefix` can never be  assigned with a `null` value here. If it were null, shouldn't it return the default value of `__` specified on L40? Which would mean that the following ternary never returns an empty string.